packages/desktop/views/account/activate/MainView.svelte

```diff
@@ -0,0 +1,69 @@
+<script lang="ts">
+    import { Button, FontWeight, Text, TextType } from 'shared/components'
+    import { MultiStep } from '../multi-step'
+    import { localize } from '@core/i18n'
+
+    interface IStep {
+        title: string
+        description: string
+        image: string
+    }
+
+    const MAIN_VIEW_STEPS: IStep[] = [
+        {
+            title: localize('views.implicit-account.steps.step1.title'),
+            description: localize('views.implicit-account.steps.step1.body'),
+            image: 'assets/illustrations/implicit-account/show-one-time-address.svg',
+        },
+        {
+            title: localize('views.implicit-account.steps.step2.title'),
+            description: localize('views.implicit-account.steps.step2.body'),
+            image: 'assets/illustrations/implicit-account/generate-mana.svg',
+        },
+        {
+            title: localize('views.implicit-account.steps.step3.title'),
+            description: localize('views.implicit-account.steps.step3.body'),
+            image: 'assets/illustrations/implicit-account/sign-transaction.svg',
+        },
+    ]
```

Comment:
I think we could change the name of the svg to `step1`, `step2` and `step3`, and so we can do this to avoid duplicating code

```suggestion
    const NUMBER_OF_STEPS = Object.keys(localize('views.implicit-account.steps')).length
    const MAIN_VIEW_STEPS: IStep[] = new Array(NUMBER_OF_STEPS).fill(null).map((_, index) => {
        console.log(index+1)
        const stepNumber = index + 1
        return {
            title: localize(`views.implicit-account.steps.step${stepNumber}.title`),
            description: localize(`views.implicit-account.steps.step${stepNumber}.body`),
            image: `assets/illustrations/implicit-account/step${stepNumber}.svg`,
        }
    })
```

wdyt?